addons/binding/org.openhab.binding.mihome/ESH-INF/thing/aqara_86sw2.xml

```diff
@@ -7,57 +7,24 @@
 		<supported-bridge-type-refs>
 			<bridge-type-ref id="bridge" />
 		</supported-bridge-type-refs>
-
 		<label>Xiaomi Aqara 2 Channel Smart Light Control</label>
 		<description>Battery powered Aqara Switch with 2 channels</description>
-
 		<channels>
-			<channel id="ch1" typeId="sw1-trg" />
-			<channel id="ch2" typeId="sw2-trg" />
-			<channel id="dual_ch" typeId="dual_sw" />
+			<channel id="ch1" typeId="system.button">
+                <label>First Button</label>
+                <description>The first pushbutton on the switch</description>
+			</channel>
+			<channel id="ch2" typeId="system.button">
+                <label>Second Button</label>
+                <description>The second pushbutton on the switch</description>
+            </channel>
+			<channel id="dual_ch" typeId="system.button">
+                <label>Both Buttons Pressed</label>
+                <description>Fires when both buttons are pressed simultaneously</description>
+            </channel>
```

Comment:
Please reformat this file using the provided code formatter or clean up settings,